config/lib/socket.io.js

```diff
@@ -85,7 +85,7 @@ module.exports = function (app, db) {
 
```

Comment:
I removed the above code (everything about Session and cookies and everything worked just fine!